src/dom/Draggable.js

```diff
@@ -75,7 +75,7 @@ export var Draggable = Evented.extend({
 
 		// If we're currently dragging this draggable,
 		// disabling it counts as first ending the drag.
-		if (L.Draggable._dragging === this) {
+		if (this._dragging === this) {

```

Comment:
I have not dived into this so deeply. Thanks for this advice! I already make a new commit...